internal/pkg/instrumentation/bpf/net/http/server/probe_test.go

```diff
@@ -48,6 +48,12 @@ func TestProbeConvertEvent(t *testing.T) {
 		Method: [8]byte{0x47, 0x45, 0x54},
 		// "/foo/bar"
 		Path: [128]byte{0x2f, 0x66, 0x6f, 0x6f, 0x2f, 0x62, 0x61, 0x72},
+		// "localhost:8080"
+		RemoteAddr: [32]byte{0x6c, 0x6f, 0x63, 0x61, 0x6c, 0x68, 0x6f, 0x73, 0x74, 0x3a, 0x38, 0x30, 0x38, 0x30, 0x0},
```

Comment:
I think the server will always set this to "IP:port": https://pkg.go.dev/net/http#Request

If not, we need to revise the storage length to accommodate the address space this could contain.